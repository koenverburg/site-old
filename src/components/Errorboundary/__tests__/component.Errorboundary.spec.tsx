// tslint:disable: no-implicit-dependencies
import {render} from '@testing-library/react'
import React from 'react'
import {ErrorBoundary} from '../'

describe('STATELESS - <ErrorBoundary />', () => {
  it.skip('will render', () => {
    const {container} = render(<ErrorBoundary fallback={<p>Loading...</p>}/>)

    expect(container).toMatchSnapshot()
  })
})
