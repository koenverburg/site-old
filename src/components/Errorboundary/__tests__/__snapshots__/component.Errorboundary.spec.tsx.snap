// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`STATELESS - <ErrorBoundary /> will render 1`] = `""`;
